src/smsfusion/_ins.py

```diff
@@ -1080,3 +1079,135 @@ def update(
         self._P_prior[:] = phi @ P @ phi.T + Q
 
         return self
+
+
+class VRU(AidedINS):
+    def __init__(
+        self,
+        fs: float,
+        x0_prior: ArrayLike,
+        P0_prior: ArrayLike,
+        err_acc: dict[str, float],
+        err_gyro: dict[str, float],
+        g: float = 9.80665,
+        nav_frame: str = "NED",
+    ):
```

Comment:
Tror ikke vi bør endre på init. x0 bør feks ha lengde 16 slik at den er kompatbel med x property. init må også være kompatibel med dump metoden. Vi kunne evt laget en alternativ constructor metode med default verdier for VRU?